avocado/utils/cpu.py

```diff
@@ -550,6 +550,23 @@ def wrap(*args, **kwargs):
     return wrap
 
 
+def lscpu():
+    # Getting the values from lscpu
```

Comment:
Can you please add docstring which will describe what return format is and what user will get. Thank you